arviz/tests/test_utils.py

```diff
@@ -119,3 +119,16 @@ def placeholder_func():
     function_results, wrapper_result = placeholder_func
     assert wrapper_result == {"keyword_argument": "A keyword argument"}
     assert function_results == "output"
+
+
+def test_numba_check():
+    test_data = np.random.rand(200).reshape(10, 20)
+
+    def placeholder(x):
+        """A simple function to test numba enhancement"""
+        dummy = 0
+        for i in range(0, x.shape[0]):
```

Comment:
In ArviZ we would normally vectorize a function like this with numpy. Do you think a plain python for loop is representative of our typical use case?

Reply:
placeholder is just a dummy method to check the numba enhancement. The kind of code in the function is one which numba always speeds up. This is just to check that that my (numba_check) function works properly. It is not a representative of any arviz function.